.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+name: Handle Dependabot PRs
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  ApproveAndMerge:
+    name: Auto approve Dependabot PRs
+    runs-on: ubuntu-latest
+    # Only run for PRs created by Dependabot - extended verification is done in the reusable workflow
+    if: github.actor == 'dependabot[bot]'
+    # These permissions are needed to approve pull requests
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Auto approve Dependabot PR
+        uses: secure-software-engineering/actions/dependabot@develop
+        with:
+          token: ${{ secrets.AUTO_MERGE_PAT }}
```

Comment:
secrets also dont work here
https://github.com/soot-oss/SootUp/actions/runs/18031767720/workflow

Reply:
my bad that seems to be something else